chai/spy.py

```diff
@@ -43,6 +54,15 @@ def side_effect(self, func, *args, **kwargs):
         self._spy_side_effect_kwargs = kwargs
         return self
 
+    def return_effect(self, func, *args, **kwargs):
```

Comment:
I am OK with ``spy_return`` but I'd just as soon keep the ``*args, **kwargs``. I think the idea that you can have a generic callback that you can parameterize when you register spies is useful.